pkg/ottl/ottlfuncs/README.md

```diff
@@ -572,6 +573,36 @@ Convert only attributes within "Record" elements
 
 - `ConvertAttributesToElementsXML(body, "/Log/Record")`
 
+### ConvertTextToElementsXML
+
+`ConvertTextToElementsXML(target, Optional[xpath])`
+
+The `ConvertTextToElementsXML` Converter returns an edited version of an XML string where all text belongs to a dedicated element.
+
+`target` is a Getter that returns a string. This string should be in XML format.
+If `target` is not a string, nil, or cannot be parsed as XML, `ConvertTextToElementsXML` will return an error.
+
+`xpath` (optional) is a string that specifies an [XPath](https://www.w3.org/TR/1999/REC-xpath-19991116/) expression that
+selects one or more elements. Content will only be converted within the result(s) of the xpath. The default is `/`.
+
+`elementName` (optional) is a string that is used for any element tags that are created to wrap content.
+The default is `"value"`.
+
+For example, `<a><b>foo</b>bar</a>` will be converted to `<a><b>foo</b><value>bar</value></a>`.
+
+Examples:
+
+Ensure all content in a document is wrapped in a dedicated element
+
+- `ConvertTextToElementsXML(body)`
+
+Use a custom naem for any new elements
+
+- `ConvertTextToElementsXML(body, "custom")`
+
+Convert only part of the document
+
+- `ConvertTextToElementsXML(body, "value", "/some/part/)`
```

Comment:
```suggestion
- `ConvertTextToElementsXML(body, "/some/part/", "value")`
```
The XPath comes before the element name in the args, so we should rearrange this.